### ISSUE TITLE
Feature Request - Ability to only switch to a dashboard screen if formula is true.

### ISSUE THREAD

**Issue by Meekstaaa** [SHWotever/SimHub#568]
Hi,

Would this be a possible future add-on? Would like to make a leaderboard screen be skipped if there aren't 20+ drivers on the grid.

Cheers :)

**Comment by Romainrob** [comment]
Hi.
To be tested but I think you can already do this with the new triggeraction function as it should allow for some dash per page (or even widget page) automation and embedded event mapping.
What if you add an new component like a transparent textbox without text so it's invisible.
And set one of the binding formula to 
if(oponnentsProp<=20, triggeraction ('theFullNameOfThe NextScreenAction'),'')
Now the limitation I see is if you used the previous screen button, it would bounce back to the next screen.

**Comment by Romainrob** [comment]
I like this idea so I did a little testing.

The automated screen change with triggeraction does work but only in one direction.

So I tried something with a widget made of two screen (one with triggeraction(nextscreen) and another with  triggeraction(previous screen)) To make it work both ways. The widget screen cycle itself when pressing next/previous screen
And it kinda works but not really. I mean the widget can get "out of sync", then it bounces you back to the screen you came from once, then works fine the next time.

If we could tell when the next and previous button are pressed. I would be possible using the triggeraction function.
Now if it's technically possible to put a condition on the whole screen, this would be much easier to use.

Here is my test dash if you want to try it:
[Screen Change Automation Test.zip](https://github.com/SHWotever/SimHub/files/4609986/Screen.Change.Automation.Test.zip)

Don't enable these 2 at once : 
![image](https://user-images.githubusercontent.com/50543913/81570083-65c9dd00-93a0-11ea-8014-2a683d2596b3.png)
